src/platform/qt/Window.h

```diff
@@ -20,13 +20,21 @@
 
 #include "ActionMapper.h"
 #include "CorePointerSource.h"
+#include "DolphinConnector.h"
+#include "FrameView.h"
+#include "GIFView.h"
 #include "InputController.h"
 #include "LoadSaveState.h"
 #include "LogController.h"
-#include "SettingsView.h"
+#include "LogView.h"
+#include "OverrideView.h"
+#include "PopupManager.h"
 #ifdef ENABLE_SCRIPTING
 #include "scripting/ScriptingController.h"
 #endif
+#include "SensorView.h"
+#include "SettingsView.h"
+#include "VideoView.h"
```

Comment:
Yes, I know that this hurts compile times. It's temporary -- I'm saving the refactor that will clean up the dependency graph for a later PR.